pages/linux/update-rc.d.md

```diff
@@ -0,0 +1,19 @@
+# update-rc.d
+
+> Install and remove service.
+
+- Install a service:
+
+`update-rc.d mysql defaults`
+
+- Forcibly remove a service:
+
+`update-rc.d -f mysql remove`
+
+- Disable a service:
+
+`update-rc.d mysql disable`
```

Comment:
Wouldn't I want to enable a service before I disable it? I'd say put enabling before disabling.